GATK/4.2.0.0/HaplotypeCaller.nf

```diff
@@ -0,0 +1,49 @@
+process HaplotypeCaller {
+    tag {"GATK HaplotypeCaller ${analysis_id} - ${interval_file.baseName}"}
+    label 'GATK_4_2_0_0'
+    label 'GATK_4_2_0_0_HaplotypeCaller'
+    container = 'broadinstitute/gatk:4.2.0.0'
+    shell = ['/bin/bash', '-euo', 'pipefail']
+
+    input:
+        tuple(val(analysis_id), path(bam_files), path(bai_files), path(interval_file))
+
+    output:
+        tuple(val(analysis_id), path("${analysis_id}.${interval_file.baseName}.vcf"), path("${analysis_id}.${interval_file.baseName}.vcf.idx"), emit: vcf_file)
```

Comment:
Split in multiple lines, like `Picard/2.26.4--hdfd78af_0/CollectArraysVariantCallingMetrics.nf`?